src/main/java/com/google/codeu/servlets/LoginServlet.java

```diff
@@ -48,4 +57,11 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     String googleLoginUrl = userService.createLoginURL("/login");
     response.sendRedirect(googleLoginUrl);
   }
+
+  private void createUserIfNew(String userEmail) {
+    if (datastore.getUser(userEmail) == null) {
+      User user = new User(userEmail, "This \"About me\" page is empty :(");
+      datastore.storeUser(user);
+    }
+  }
```

Comment:
This is the bulk of the important stuff in this PR - I create a user on login if they do not already exist